src/main/java/pl/ateam/disasteralerts/user/dto/UserLoginDTO.java

```diff
@@ -1,6 +1,14 @@
 package pl.ateam.disasteralerts.user.dto;
 
+import jakarta.validation.constraints.*;
+
 public record UserLoginDTO(
+        @Email @NotBlank
+        @Max(255)
         String email,
+        String location,
+        @NotBlank
+        @Size(min=6,max=25)
+        @Pattern(regexp = "Password must contain at least one digit, one lowercase letter, one uppercase letter, one special character, and be between 8 and 16 characters long")
```

Comment:
```suggestion
        @Pattern(regexp = "",  message = "Password must contain at least one digit, one lowercase letter, one uppercase letter, one special character, and be between 8 and 16 characters long")
```
Brakuje regex, lub można też message dodać do size
```
@Size(min=6, max=25, message = "Password must contain at least one digit, one lowercase letter, one uppercase letter, one special character, and be between 8 and 16 characters long")
```